scripts/rspec/rspec2resx.ps1

```diff
@@ -71,9 +71,11 @@ $roslynLanguageMap = @{
 }
 
 function FindResGen(){
+    Write-Host "Finding ResGen.exe path"
     foreach ($SDK in $SupportedSDKs){
         $Ret = "${Env:ProgramFiles(x86)}\\Microsoft SDKs\\Windows\\v10.0A\\bin\\NETFX ${SDK} Tools\\ResGen.exe"
```

Comment:
could you also print the folder structure inside `"${Env:ProgramFiles(x86)}\\Microsoft SDKs\\Windows\\v10.0A\\bin\\` ? 

Because #2763 says that the script doesn't work with only 4.8 installed, but I see at line 35 that `$SupportedSDKs` has
```
$SupportedSDKs = @("4.8", "4.7.2", "4.6.1")
```

so I expect that 4.8 has some different structure?

Reply:
and anyway, to see if there's a corelation between failures and machine setup

Reply:
I doubt that there is a problem with the folder structure. I've checked on my pc and it's the following:
```
C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.7.2 Tools\ResGen.exe
C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.1 Tools\ResGen.exe
C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\ResGen.exe
```

If there will be a problem with this, we would see the following error and will know to investigate more:
> You need to install the .NET Framework $($SupportedSDKs[0]) SDK before using this script.

I think that dumping the folder structure will pollute too much the logs. We can add this later if we notice the above message.